src/Module/Api/Mastodon/Accounts/Lists.php

```diff
@@ -37,16 +37,16 @@ class Lists extends BaseApi
 	 */
 	protected function rawContent(array $request = [])
 	{
-		self::checkAllowedScope(self::SCOPE_READ);
+		$this->checkAllowedScope(self::SCOPE_READ);
 		$uid = self::getCurrentUserID();
 
 		if (empty($this->parameters['id'])) {
-			DI::mstdnError()->UnprocessableEntity();
+			$this->logAndJsonError(422, $this->errorFactory->UnprocessableEntity());
```

Comment:
Seeing all these `$this->logAndJsonError(<HTTP Error Number>, $this->errorFactory-><HTTP Error Description>());` lines, I'm asking myself, why the error number and and the error message is split. Why not having a single function that provides both the error number and the error description?

Reply:
In this PR, I've tried to stick as close to the original feature while solving the fatal error. Anything is possible beyond that, but given the existing size of this diff, I'd rather do it in a later PR.

For example, the HTTP error code could be added to the Mastodon error entity, which would remove the need for the first parameter.

Reply:
Hmm, okay.